pkg/generator.go

```diff
@@ -201,6 +201,9 @@ func (g *Generator) mockName() string {
 		if ast.IsExported(g.iface.Name) {
 			return "Mock" + g.iface.Name
 		}
+		if !ast.IsExported(g.iface.Name) && g.Exported {
```

Comment:
Better yet, let's just say `if g.Exported { }`.

Reply:
you're right, fixed.